core/pta/tests/misc.c

```diff
@@ -21,6 +25,226 @@
  */
 #define LOG(...)
 
+#ifdef CFG_IT_NOTIF_TEST
+static struct notif_it test_it_notif[TEST_IT_NOTIF_CNT];
+/* Helper to release registered resource in case of error */
+static bool test_it_notif_registered[TEST_IT_NOTIF_CNT];
+
+static unsigned int idx2id(unsigned int test_idx)
```

Comment:
So now we have interrupt indexes too? This is starting to get really confusing.

Reply:
sorry... I'll try to simplify that.